packages/ceph/ceph.20240106/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+maintainer: "Ahrefs Pte Ltd <github@ahrefs.com>"
+authors: [ "Ahrefs Pte Ltd <github@ahrefs.com>" ]
+license: "LGPL-2.1-only WITH OCaml-LGPL-linking-exception"
+homepage: "https://github.com/ahrefs/ocaml-ceph"
+dev-repo: "git+https://github.com/ahrefs/ocaml-ceph.git"
+bug-reports: "https://github.com/ahrefs/ocaml-ceph/issues"
+tags: [ "org:ahrefs" "clib:stdc" "clib:cephfs"  ]
+build: [
+  ["dune" "subst"] {dev}
+  ["dune" "build" "-p" name "-j" jobs "@install" "@doc" {with-doc}]
+]
+depends: [
+  "ocaml"
+  "dune" {>= "2.0"}
+  "odoc" {with-doc}
+  "ctypes"
+]
+depexts: [
```

Comment:
```suggestion
conflicts: [
  "ocaml-option-bytecode-only"
]
depexts: [
```

It fails to build the runtime system on all the CI machine with bytecode only compiler